src/dsp.rs

```diff
@@ -193,15 +194,15 @@ fn fast_resampling(
     // Save expanded and filtered signal if we need to export that step
     let mut expanded_filtered = if context.export_resample_filtered  {
         // Very likely to overflow usize on 32 bits systems
-        match interpolated_len.to_usize() {
-            Some(l) => Vec::with_capacity(l),
-            None => {
+        match usize::try_from(interpolated_len) {
+            Ok(l) => Vec::with_capacity(l),
+            Err(_) => {
                 error!("Expanded filtered signal can't fit in memory, skipping step");
-                Vec::with_capacity(0) // Not going to be used
+                Vec::new() // Not going to be used
```

Comment:
Just to remember myself and fix it: I need to set `context.export_resample_filtered` to `false` when this happens